tests/unit/s2n_client_psk_extension_test.c

```diff
@@ -365,8 +352,10 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_stuffer_alloc(&wire_identities_in, sizeof(wire_identites)));
             EXPECT_SUCCESS(s2n_stuffer_write_bytes(&wire_identities_in, wire_identites, sizeof(wire_identites)));
 
-            EXPECT_OK(s2n_client_psk_recv_identity_list(conn, &wire_identities_in));
+            EXPECT_ERROR_WITH_ERRNO(s2n_client_psk_recv_identity_list(conn, &wire_identities_in),
+                                      S2N_ERR_VALID_PSK_IDENTITY_NOT_FOUND);
             EXPECT_NULL(conn->psk_params.chosen_psk);
+            EXPECT_EQUAL(conn->psk_params.chosen_psk_wire_index, 0);
```

Comment:
Do we make any guarantees about the value of chosen_psk_wire_index if there's no match? Why would it need to be 0?